### PR TITLE
You can only behead by attacking neck

### PR DESCRIPTION
**Pull request by panvxv**
## About The Pull Request
Change so players need to go for neck - harder to hit part - to behead someone.

## Why It's Good For The Game
Beheading was too easy and required little weapon skill and stats.

### CODE REVIEW

**Comment by CVSthePharmacy** [comment]
if you do this then axes and maces will be the new meta, as axes can embed in heads for an instakill

**Comment by TheManWithNoHands** [comment]
> if you do this then axes and maces will be the new meta, as axes can embed in heads for an instakill

Yeah but you can counter axe to the head by a helmet. Just as you can counter beheading by sword with neckguard with this.
Maces are already meta for people that know what they are doing. You can knock someone out for like 15 seconds if you hit their head from behind. Or just onehit kill em even with a helmet on. Cutting someones head off as top combat strategy is too cheesy. A rock paper siccors thing is healthier.

**Comment by Gyrandola** [comment]
> > if you do this then axes and maces will be the new meta, as axes can embed in heads for an instakill
> 
> Yeah but you can counter axe to the head by a helmet. Just as you can counter beheading by sword with neckguard with this. Maces are already meta for people that know what they are doing. You can knock someone out for like 15 seconds if you hit their head from behind. Or just onehit kill em even with a helmet on. Cutting someones head off as top combat strategy is too cheesy. A rock paper siccors thing is healthier.

arent swords still worse than maces this way though.

**Comment by Wintermote** [comment]
I don't think this is a good solution to the problem

Instead, I think attacks on the head need to have a wildly lower chance to hit unless someone is laying prone, and a higher chance if they are prone + immobilized. So that aiming for the head instead of chest is a very risky move in a fight, rather than what everyone does. 

That would also help the problem with mace stuns.

**Comment by panvxv** [comment]
> if you do this then axes and maces will be the new meta, as axes can embed in heads for an instakill

axes actually don't have insta kill on head hit, swords have as stab crit to the face insta kill.

**Comment by CVSthePharmacy** [comment]
> > if you do this then axes and maces will be the new meta, as axes can embed in heads for an instakill
> 
> axes actually don't have insta kill on head hit, swords have as stab crit to the face insta kill.

wdym, to the eyes or nose?

**Comment by CVSthePharmacy** [comment]
> > if you do this then axes and maces will be the new meta, as axes can embed in heads for an instakill
> 
> axes actually don't have insta kill on head hit, swords have as stab crit to the face insta kill.

axes have embed chance that instakill

**Comment by panvxv** [comment]
> > > if you do this then axes and maces will be the new meta, as axes can embed in heads for an instakill
> > 
> > 
> > axes actually don't have insta kill on head hit, swords have as stab crit to the face insta kill.
> 
> axes have embed chance that instakill

only on chest, not on head

**Comment by CVSthePharmacy** [comment]
and on head

**Comment by TheManWithNoHands** [comment]
> arent swords still worse than maces this way though.

That maces dont have a counter really, is a diffrent problem.

**Comment by bobjension** [comment]
godly PR

**Comment by ChungusGamer666** [comment]
Godly pr

File: code/modules/surgery/bodyparts/dismemberment.dm [hunk 1]
```diff
@@ -14,12 +14,13 @@
...
 //Dismember a limb
-/obj/item/bodypart/proc/dismember(dam_type = BRUTE)
+/obj/item/bodypart/proc/dismember(dam_type = BRUTE, zone_precise)
 	if(!owner)
 		return FALSE
 	var/mob/living/carbon/C = owner
 	if(!dismemberable)
-		return FALSE
+		if(zone_precise != BODY_ZONE_PRECISE_NECK)
+			return FALSE
```
Comment (ChungusGamer666) [inline code/modules/surgery/bodyparts/dismemberment.dm:23]: This check is absolutely insane. DO NOT DO THIS, dismemberable is called that for a reason.

Add another if check instead below this, if(body_zone == BODY_ZONE_HEAD && (zone_precise != BODY_ZONE_PRECISE_NECK)) return FALSE

**Review by ChungusGamer666 — CHANGES_REQUESTED** [review]

**Comment by Wintermote** [comment]
There is a bug (already mentioned in discord) with the guillotine, which appears to be doing head damage instead of neck damage and as such will never actually decapitate anyone.

**Comment by CVSthePharmacy** [comment]
> > arent swords still worse than maces this way though.
> 
> That maces dont have a counter really, is a diffrent problem. Problem with decaps, they are rather easy to do. And they perma kill you. Any other way you die and you can be revived. As long as the head stays on. And to be perma killed from a oneshot from 100% to 0 health, is something that only can cause frustration in the beheaded in question. So macking it so, that you actually need to hit the neck, to cut through the neck, is kinda good.

ok but you see the thing is blackstone combat is quite something, you cannot die via getting maced in the chest, you can have a morbillion brute damage and not die, it's only critical hits to the head with a mace that get you

**Pull request merged**